packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -52,6 +56,10 @@ const nextServerlessLoader: loader.Loader = function() {
   const escapedBuildId = escapeRegexp(buildId)
   const pageIsDynamicRoute = isDynamicRoute(page)
 
+  const encodedPreviewProps = devalue(
+    JSON.parse(previewProps) as __ApiPreviewProps
+  )
```

Comment:
What's the benefit to decoding and then re-encoding here? It's a pretty simple JSON string so `devalue` doesn't provide any benefits right?

Reply:
Devalue would reduce the object size if this was ever made more complex in the future. It's a premature optimization, I can remove if you feel strongly about it.